packages/core-sdk/src/abi/json/LicenseRegistry.abi.ts

```diff
@@ -1,4 +1,78 @@
 export default [
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: true,
+        internalType: "address",
+        name: "creator",
+        type: "address",
+      },
+      {
+        indexed: true,
+        internalType: "address",
+        name: "receiver",
+        type: "address",
+      },
+      {
+        indexed: true,
+        internalType: "uint256",
+        name: "licenseId",
+        type: "uint256",
+      },
+      {
+        indexed: false,
+        internalType: "uint256",
+        name: "amount",
+        type: "uint256",
+      },
+      {
+        components: [
+          {
+            internalType: "uint256",
+            name: "policyId",
+            type: "uint256",
+          },
+          {
+            internalType: "address[]",
+            name: "licensorIpIds",
+            type: "address[]",
+          },
+        ],
+        indexed: false,
+        internalType: "struct Licensing.License",
+        name: "licenseData",
+        type: "tuple",
+      },
+    ],
+    name: "LicenseMinted",
+    type: "event",
+  },
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: true,
+        internalType: "address",
+        name: "caller",
+        type: "address",
+      },
+      {
+        indexed: true,
+        internalType: "address",
+        name: "ipId",
+        type: "address",
+      },
+      {
+        indexed: true,
+        internalType: "uint256",
+        name: "policyId",
+        type: "uint256",
+      },
+    ],
+    name: "PolicyAddedToIpId",
+    type: "event",
```

Comment:
What about LicenseBurnt?

Reply:
Actually there isn't a licenseBurnt in the smart contract yet